docs/pages/configuration/_partials/functions/start_dev/example.mdx

```diff
@@ -0,0 +1,16 @@
+#### Example:
+
+```yaml
+pipelines:
+  dev: |-
+     echo "Welcome to your custom dev pipeline!"
+
+     # Exchange dev config here (use --set-string if you want to force the field to be a string e.g. when setting labels to "true")
+     start_dev myDevConfig --set myConfigField=value --set-string key1=value1
+    
+     # Use --from to create a new config based on an existing config
+     start_dev myOtherDevConfig --from myDevConfig
+
+     # Use --from-file when you want to load the config for the image from a file
```

Comment:
This comment seems inaccurate since there is no image in this config

Reply:
Correct! I've fixed this!